src/yadism/coefficient_functions/coupling_constants.py

```diff
@@ -477,6 +479,10 @@ def __repr__(self):
         """Return string representation."""
         return "CKM(" + str(self.m).replace("\n", "") + ")"
 
+    def __eq__(self, other) -> bool:
+        """Equal method."""
+        return (self.m == other.m).all()
```

Comment:
@giacomomagni that's not good (float and `==`) - it should rather be `np.assert_all_close`

Reply:
sure feel free to adjust it, I added this mostly to make a test not because we really need it

Reply:
I think just `np.allclose`, but you're right. Then again, the input is always the exact same and we don't do any operations on it (or the same on all, namely taking the square only once), so comparing two should really always give the same result even accounting for finite precision due to floats.